setup.py

```diff
@@ -0,0 +1,22 @@
+"""Set up file for imjoy-elfinder."""
+import json
+from pathlib import Path
+
+from setuptools import setup, find_packages
+
+PROJECT_DIR = Path(__file__).parent.resolve()
+PACKAGE_JSON = json.loads((PROJECT_DIR / "package.json").read_text())
+
+
+setup(
+    name="imjoy-elfinder",
```

Comment:
I was thinking that we should rename `jupyter-elfinder` to `imjoy-elfinder`, so maybe we can rename this to something else? How about `imjoy-elfinder-client`?

Reply:
Yes.